### DISCUSSION TITLE
Manage multiple working git trees

### DISCUSSION THREAD

**Discussion by andy5995** [mesonbuild/meson#10807] [General]
I haven't tried this much yet, but it seems like it could be pretty useful in some cases.

[git-worktree - Manage multiple working trees](https://git-scm.com/docs/git-worktree)

>A git repository can support multiple working trees, allowing you to check out more than one branch at a time. With git worktree add a new working tree is associated with the repository, along with additional metadata that differentiates that working tree from others in the same repository. The working tree, along with this metadata, is called a "worktree".

[YouTube Video | Git Worktree Tutorial | The Most Underrated Git Command?](https://www.youtube.com/watch?v=4_p1OdLeDLE)

**Comment by eli-schwartz** [comment]
I indeed find it very useful. Not specifically for Meson projects, but certainly including them.
